Dot.cs

```diff
@@ -0,0 +1,13 @@
+﻿class Dot
+{
+    public float coordX0, coordY0;
+    public Dot(float coordX, float coordY)
+    {
+        this.coordX0 = coordX;
+        this.coordY0 = coordY;
+    }
+    public virtual bool IsFigure(float coordX, float coordY)
+    {
+        return (coordX == coordX0) && (coordY == coordY0); //(coordX == X0) && (coordY == Y0)

```

Comment:
If you write this comment you think, that people who would read your code can have some questions to this line. Do you think you can have more readable variable names in that case?